canonical-web-and-design/ubuntu.com
# Fix blog-cards not display variably coloured top strip

## Done

- Fix blog-cards not display variably coloured top strip

## QA

- Go to /blog and check that the blog cards are display a different coloured top strip depending on the catergory

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11787
